apps/files/decorators.py

```diff
@@ -18,21 +18,22 @@
 
 
 def allowed(request, file):
-    allowed = acl.check_addons_reviewer(request)
-    if not allowed:
-        try:
-            addon = file.version.addon
-        except ObjectDoesNotExist:
-            raise http.Http404
-
+    try:
+        addon = file.version.addon
+    except ObjectDoesNotExist:
+        raise http.Http404
```

Comment:
It was already there, so don't worry about it, but do we still need that check ? Cascading deletes + the fact that we are using unfiltered for relations should mean we no longer need it. Can't hurt to have it I guess.